source/Willow-Bootstrap.package/BootstrapDropdownBuilder.class/instance/addActionLabeled.executing..st

```diff
@@ -3,6 +3,8 @@ addActionLabeled: aRenderable executing: anActionBlock
 
 	| anchor |
 
-	anchor := AsynchronicAnchorWebView labeled: aRenderable.
+	anchor := supplier
```

Comment:
Please revert this change, the markup for this components specifically needs a plain anchor.